glotaran/model/dataset_model.pyi

```diff
@@ -16,6 +16,10 @@ def create_dataset_model_type(properties: dict[str, Any]) -> type[DatasetModel]:
 class DatasetModel:
 
     label: str
+    clp_guidance: str | None
+    overwrite_model_dimension: str | None
+    overwrite_global_dimension: str | None
```

Comment:
Change overwrite to override.

Reply:
Fixed in [7f565aa](https://github.com/glotaran/pyglotaran/pull/983/commits/7f565aac69c6433bfdc04444c3b2f0dd1997b31f)